src/dialect/sqlite/sqlite-driver.ts

```diff
@@ -92,8 +92,24 @@ class SqliteConnection implements DatabaseConnection {
     }
   }
 
-  async *streamQuery<R>(): AsyncIterableIterator<QueryResult<R>> {
-    throw new Error("Sqlite driver doesn't support streaming")
+  async *streamQuery<R>(
+    compiledQuery: CompiledQuery,
+    _chunkSize: number
```

Comment:
can we implement it with usage of `chunkSize`?

Reply:
It was a little unclear what `chunkSize` is meant for - is this for the "#" of rows returned at once? Or is it meant in association with the `chunkSize` option for streams.

In the docstring here, it says it's only supported in PG: https://github.com/kysely-org/kysely/blob/0982d61a59d6dccaaf2a026cad82a77b843be7ac/src/query-executor/query-executor.ts#L59-L63

Though it looks like it's also used in mssql: https://github.com/kysely-org/kysely/blob/0982d61a59d6dccaaf2a026cad82a77b843be7ac/src/dialect/mssql/mssql-driver.ts#L181-L191

Both of those seem to be using it in association with the underlying stream, and it looks to be unused in `mysql`:

https://github.com/kysely-org/kysely/blob/0982d61a59d6dccaaf2a026cad82a77b843be7ac/src/dialect/mysql/mysql-driver.ts#L181-L192

So since the iterator returned from better-sqlite isn't actually a stream, I decided to ignore it.